mgc/discrim/discrimOneSample.py

```diff
@@ -34,9 +34,7 @@ class oneSample(discriminabilityTest):
     """
 
     def __init__(self):
-        discriminabilityTest.__init__(self)
-
-
+        DiscriminabilityTest.__init__(self)
 
     def test(self, X, Y, is_dist = False, remove_isolates=True, reps=1000, workers=-1):
```

Comment:
make `x` and `y` lowercase by convention everywhere. Don't need `is_dist` here as mentioned before, you can determine that yourself. `remove_isolates` should probably be in the `__init__`